src/components/AddIcon/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+export default function AddIcon({ size }) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      class={`h-${size} w-${size}`}
```

Comment:
This condition is wrong. 
```suggestion
      className={`h-${size} w-${size}`}
```